machinist/test/test_fsm.py

```diff
@@ -868,7 +868,6 @@ def test_terminalLogging(self, logger):
                     u"action_type": u"fsm:initialize",
                     u"action_status": u"succeeded",
                     u"task_uuid": logger.messages[0][u"task_uuid"],
-                    u"task_level": u"/",
```

Comment:
Maybe something about `task_level` should still be logged here?  Or ... `action_type`, `action_status`, and `task_uuid` are sufficient to prove it's the right event and if `task_level` isn't what we expect it must be a bug in eliot?